### PR TITLE
Elaborating exiting pygame applications

### PR DESCRIPTION
**Pull request by blankRiot96**
Covered a few different styles of exiting a `pygame` application and the benefits of each.

### CODE REVIEW

**Comment by oddbookworm** [comment]
This looks good, but there is one other way that I've used before:
If your game loop is in a function like `main`, then just `return` after the `QUIT` event works too. Not sure that one should be mentioned though

**Review by oddbookworm — DISMISSED** [review]
LGTM

**Comment by blankRiot96** [comment]
Had mentioned on discord too: You are right and that is indeed a valid way to exit the application! But I don't think you would see it in a more structural larger game(mostly because there is some complexity involved to work around it for it to be used) and I want to build the foundation for that! 
But it is definitely a valid thing and it may have been worth mentioning if I had a `main` function already. Unfortunately I did not create a `main` function to contain my code to keep it as simple as possible

File: intro/setup/index.html [hunk 1]
```diff
@@ -347,16 +384,19 @@ <h2><a id="final-result" class="js"></a>Final Result</h2>
 WIN_SIZE = (420, 250)
 screen = pygame.display.set_mode(WIN_SIZE, flags=pygame.SCALED)
 
+is_running = True
 while True:
```
Comment (pycoinfu) [inline intro/setup/index.html:388]: Instead of `while True` it should be `while is_running`, otherwise `is_running` isn't doing anything

**Review by pycoinfu — CHANGES_REQUESTED** [review]

**Comment by oddbookworm** [comment]
#61 replaced this PR, so I'm closing it

**Pull request closed without merging**